cmd/coap/main.go

```diff
@@ -35,9 +36,10 @@ const (
 )
 
 type config struct {
-	LogLevel  string `env:"MF_INFLUX_READER_LOG_LEVEL"  envDefault:"info"`
-	BrokerURL string `env:"MF_BROKER_URL"               envDefault:"nats://localhost:4222"`
-	JaegerURL string `env:"MF_JAEGER_URL"               envDefault:"localhost:6831"`
+	LogLevel   string `env:"MF_INFLUX_READER_LOG_LEVEL"  envDefault:"info"`
+	BrokerURL  string `env:"MF_BROKER_URL"               envDefault:"nats://localhost:4222"`
+	JaegerURL  string `env:"MF_JAEGER_URL"               envDefault:"localhost:6831"`
+	InstanceID string `env:"MF_COAP_ADAPTER_INSTANCE_ID"   envDefault:""`
```

Comment:
envDefault alginement suggestion 


```suggestion
	LogLevel   string `env:"MF_INFLUX_READER_LOG_LEVEL"    envDefault:"info"`
	BrokerURL  string `env:"MF_BROKER_URL"                 envDefault:"nats://localhost:4222"`
	JaegerURL  string `env:"MF_JAEGER_URL"                 envDefault:"localhost:6831"`
	InstanceID string `env:"MF_COAP_ADAPTER_INSTANCE_ID"   envDefault:""`
```